docs/src/app/sandbox.tsx

```diff
@@ -1,154 +1,198 @@
-                        <Button
-                          variant="ghost"
-                          rightIcon={ <Icon as={ LinkDuo } /> }
-                        >
-                          Contribute
-                        </Button>
-                      </Link>
-                    </HStack>
-                  </Grid>
-                  <Grid
-                    gridTemplateColumns={ pageProportions }
-                    color="text.default"
-                    w="full"
-                    position="relative"
+                    </DrawerHeader>
+                    <DrawerBody sx={ { bgColor: 'background.default' } }>
+                      <MainMenu menuItems={ routes } />
+                      <SubMenu mainRoutes={ routes } />
+                    </DrawerBody>
+                  </DrawerContent>
+                </Drawer>
+
+                <Flex
+                  pt="64px"
+                  minH="100vh"
+                  direction={ { md: 'row' } }
+                  flex={ 1 }
+                  w={ { base: '100%', md: 'auto' } }
+                  maxW={ { base: '100vw', md: 'none' } }
+                  overflowX={ { base: 'hidden', md: 'initial' } }
+                >
+                  <Box
+                    w="20rem"
+                    h="100%"
+                    px={ 2 }
+                    overflowY="auto"
+                    position="fixed"
+                    display={ { base: 'none', md: 'block' } }
+                    left="0"
+                    top="100"
+                    zIndex="sticky"
+                    bgColor="bg.base"
                   >
-                    <GridItem>
-                      <Stack
-                        w="20rem"
-                        position="fixed"
-                        bgColor="bg.layer"
-                        p="2"
-                      >
-                        <MainMenu menuItems={ routes } />
-                        <SubMenu mainRoutes={ routes } />
-                      </Stack>
-                    </GridItem>
-                    <GridItem>
-                      <Center>
-                        <Routing
-                          fallback={ head(routes)?.path }
-                          routes={ routes }
-                        />
-                      </Center>
-                    </GridItem>
-                    <GridItem />
-                  </Grid>
-                </Stack>
-              </Center>
+                    <MainMenu menuItems={ routes } />
+                    <Divider my={ 5 } w="92%" />
+                    <SubMenu mainRoutes={ routes } />
+                  </Box>
+
+                  <Flex
+                    flex="1"
+                    pl={ { md: '21rem' } }
+                    pt={ 4 }
+                    boxSizing="border-box"
+                  >
+                    <VStack spacing={ 4 }>
+                      <Routing fallback={ head(routes)?.path } routes={ routes } />
+                    </VStack>
```

Comment:
If you try to go to https://deploy-preview-180--luminous-medovik-367cf4.netlify.app/tokens/global-tokens its gonna look really weird becuase the tokens jsoncrack does not take up entire width, to fix this put w="full" on `<Flex> `and `<VStack>` here